core/src/main/java/feast/core/service/JobCoordinatorService.java

```diff
@@ -99,16 +97,11 @@ public void Poll() throws InvalidProtocolBufferException {
       Store store = Store.fromProto(storeSpec);
 
       for (Subscription subscription : store.getSubscriptions()) {
-        var featureSetSpecs =
-            specService
-                .listFeatureSets(
-                    ListFeatureSetsRequest.Filter.newBuilder()
-                        .setFeatureSetName(subscription.getName())
-                        .setFeatureSetVersion(subscription.getVersion())
-                        .setProject(subscription.getProject())
-                        .build())
-                .getFeatureSetsList();
-        featureSets.addAll(featureSetsFromProto(featureSetSpecs));
+        List<FeatureSet> featureSetsForSub =
+            featureSetRepository.findAllByNameLikeAndProject_NameLikeOrderByNameAsc(
+                subscription.getName().replace('*', '%'),
+                subscription.getProject().replace('*', '%'));
+        featureSets.addAll(featureSetsForSub);
```

Comment:
Not sure of the benefits of bypassing `SpecService` here. 
If we centralize listing of FeatureSets through `ListFeatureSets`,  changes to `listFeatureSets()` can propagate upstream (ie `Subscriptions`, `JobService`, etc.).